server.js

```diff
@@ -21,9 +21,55 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  
+  const EntryPage = {
+    Welcome: "It's the possibility of having a dream come true that makes life interesting. - The Alchemist",
+    Endpoints: [
+      {
+        "/booksData": "Display all books",
+        "/booksData/title/:title": "Look for a specific title",
+        "/booksData/average_rating/": "Sort on rating (Higest to lowest)",
+        "/booksData/num_pages/": "Find the book with most pages",
+      },
+    ],
+  };
+  res.send(EntryPage);
 });
 
+app.get('/booksData', (req, res) => {
+  res.status(200).json({
+    data: booksData,
+    success: true,
+  })
+  })
+
+app.get("/booksData/title/:title", (req, res) => {
+  const { title } = req.params;
+
+  const booksByTitle = booksData.filter((book) => book.title.toUpperCase().includes(title.toUpperCase()) //.includes() for defining whether the array contains the specified element or not. 
+    );
+      res.status(200).json({
+        data: booksByTitle,
+        success: true,
+    })
+});
+
+app.get("/booksData/average_rating/", (req, res) => {
+  const { average_rating } = req.params;
+
+  const booksByRating = booksData.sort((a, b) => b.average_rating - a.average_rating) //.sort b-a sorts the elements of an array as strings in an ascending order. a (the first element for comparison. b = the second element for comparison.
```

Comment:
I like that you have implemented this sorting of ratings from highest to lowest. Nice work!